### ISSUE TITLE
Rework footer

### ISSUE THREAD

**Issue by szarnyasg** [duckdb/duckdb-web#1285]
I'd like to rework the "Follow us" section in the footer (`_include/footer.html`) as follows:

```html
<h6>Community</h6>
<div class="icons">
	<a href="https://x.com/duckdb" target="_blank"><span class="duckdbsymbol">&#xe045;</span> X (Twitter)</a>
	<a href="https://discord.duckdb.org" target="_blank"><span class="duckdbsymbol">&#xe055;</span> Discord</a>
	<a href="https://github.com/duckdb/duckdb" target="_blank"><span class="duckdbsymbol">&#xe050;</span> GitHub</a>
</div>
```

I'd like to spell out the names of the platforms, especially Discord. This would allow us to remove the Discord ribbon from the top of the site (see PR #1158).

X should have the new X logo: https://about.twitter.com/en/who-we-are/brand-toolkit

Can you please apply this HTML and ensure that the result look correct?

**Comment by Mause** [comment]
I think `X (Twitter)` would be best?

**Comment by szarnyasg** [comment]
Fully agree, people will take years to adjust, and every news piece I've read refers to the platform as "X, formerly known as Twitter". I edited the HTML.

**Comment by jonathanauch** [comment]
https://github.com/duckdb/duckdb-web/pull/1294

**Comment by szarnyasg** [comment]
Is it possible to have both the symbols and the texts? If it looks bad, then let's go for text-only.

**Comment by jonathanauch** [comment]
I would go for text only. Also not to happy with the length of the name "X" – but i guess that's how it is from now on

**Comment by szarnyasg** [comment]
Fixed by #1294.
